### ISSUE TITLE
Pagination styling is not consistent with CMS patterns

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/silverstripe-asset-admin#1157]
Looks like pagination styling has been broken in 4.7.0 Beta-1.

![image](https://user-images.githubusercontent.com/24258161/98314677-5d92e780-203b-11eb-986c-834f66d2c105.png)

cc @silverstripeux

**Comment by brynwhyman** [comment]
Might be related to the Reactstrap 8 upgrade?

**Comment by maxime-rainville** [comment]
Could be related to https://github.com/silverstripe/silverstripe-asset-admin/pull/1095

**Comment by sachajudd** [comment]
Looks like this is due to Chrome. It looks like this in Firefox:
![image](https://user-images.githubusercontent.com/24258161/98491242-15710080-2299-11eb-9860-c736c430ea42.png)

Since this isn't a regression we can mark this as lower priority, I'll add some designs to make this more consistent with the patterns we have now @maxime-rainville.

**Comment by brynwhyman** [comment]
Noting that to resolve this properly we'd likely need to create a new component specific for the Files area. Using the GridField component (the other screenshot) isn't going to make sense here.

**Comment by sachajudd** [comment]
Yup, agreed. I didn't get the chance to update this after talking with the team.

**Comment by ChloeSartorelli** [comment]
Unclear how this should look, it's looked like this so long that this is the new normal. Closing - if we ever change this it's probably gonna be part of a wider reskin.
